pkg/aks/create_test.go

```diff
@@ -0,0 +1,58 @@
+package aks
+
+import (
+	"errors"
+
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2019-10-01/resources"
+	"github.com/Azure/go-autorest/autorest/to"
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/rancher/aks-operator/pkg/aks/services/mock_services"
+	aksv1 "github.com/rancher/aks-operator/pkg/apis/aks.cattle.io/v1"
+)
+
+var _ = Describe("CreateResourceGroup", func() {
+	var (
+		mockCtrl          *gomock.Controller
```

Comment:
IMO I prefer variables to be written out so it's completely readable, like `mockController` and `mockResourceGroupClient`